src/minisat/mtl/BoxedVec.h

```diff
@@ -21,7 +21,7 @@ OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWA
 #define BoxedVec_h
 
 #include <cstdlib>
-#include <cassert>
```

Comment:
I don't think we are using this file, no? So it should probably not be changed.

Reply:
Done.